src/Service/JwtService.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Domain.Models;
+using Domain.Options;
+using Domain.Responses;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace Application.Services;

```

Comment:
Se for dar merge na PR anterior, atualizar da main após.
Senão adequar o namespace para Service.